arrows/core/depth_utils.cxx

```diff
@@ -329,6 +331,124 @@ compute_pixel_to_world_scale(kwiver::vital::vector_3d const& minpt,
   return scale / count;
 }
 
+//*****************************************************************************
+
+/// Gather images and masks corresponding to cameras from video sources
+int gather_depth_frames(
```

Comment:
This appears to add the same content as the [Cameras by angle PR](https://github.com/Kitware/kwiver/pull/1152/files#diff-c95d1252b1f20945632afad074033a6fa970cd7d82e06a782927c8a7c1110800R337). I'm guessing this was intentional, but just wanted to confirm.

Reply:
yes, this branch requires some of the changes in #1152, so they are duplicated for now.  I made a note about this above.  I probably should have just waited to push this until #1152 was merged.  I'll clean this up once #1152 merges.

Reply:
done